control_plane/src/endpoint.rs

```diff
@@ -147,11 +148,12 @@ impl ComputeControlPlane {
 
     /// Create a JSON Web Key Set. This ideally matches the way we create a JWKS
     /// from the production control plane.
-    fn create_jwks_from_pem(pem: Pem) -> Result<JwkSet> {
-        let document = pkcs8::Document::from_der(&pem.into_contents())?;
+    fn create_jwks_from_pem(pem: &Pem) -> Result<JwkSet> {
+        let spki: SubjectPublicKeyInfoRef = SubjectPublicKeyInfo::from_der(pem.contents())?;
+        let public_key = spki.subject_public_key.raw_bytes();
 
         let mut hasher = Sha256::new();
-        hasher.update(&document);
+        hasher.update(public_key);
         let key_hash = hasher.finalize();
```

Comment:
Hmm, so this is used as the key_id, i.e. the `kid` field. Is that a neon convention, does the client sending the tokens do that too? Or did we just need any arbitrary value? Can we leave it None?

Reply:
Yes, this is the `kid` that the control plane sends. I asked @mtyazici if we should send something human readable and searchable, but he didn't seem to think it was needed. Perhaps he can clarify here.